src/index.html

```diff
@@ -10,13 +10,88 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="bar">
+          <a
+            href="#"
+            class="bar__link is-active"
+          >
+            Apple
+          </a>
```

Comment:
The ul tag must be inside the lш tag, so you need to fix it and class everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
            >
              Apple
            </a>
          </li>
```